.gitignore

```diff
@@ -1,6 +1,5 @@
 .idea
 node_modules
-<<<<<<< HEAD
 neo4j
```

Comment:
I think neo4j can be removed as well since it is not used in this project (anymore). It was used in the past to visualize the generated coupling relationships but has been remoed.